client/js/components/ThreadSection.js

```diff
@@ -23,8 +23,8 @@ var ThreadSection = React.createClass({
   },
 
   componentDidMount: function() {
-    console.log('ThreadSection addChangeListener');
     ThreadStore.addChangeListener(this._onChange);
+    ThreadActionCreators.clickThread('0');
```

Comment:
We might need a `if (ThreadStore.getCurrentThread()) {}` around this incase it fires more than just the first time.